src/App.js

```diff
@@ -1,26 +1,12 @@
 import React from 'react';
-import { Link, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
-import Home from './components/Home';
-import Quote from './components/Quote';
-import './css/nav.css';
+import Quotes from './components/Quotes';
 
 export default function App() {
   return (
-    <>
-      <nav>
-        <h1 className="nav-bar-h1">Math Magicians</h1>
-        <ul className="nav-link">
-          <li><Link to="/">Home</Link></li>
-          <li><Link to="/calculator">Calculator</Link></li>
-          <li><Link to="/quote">Quote</Link></li>
-        </ul>
-      </nav>
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/calculator" element={<Calculator />} />
-        <Route path="/quote" element={<Quote />} />
-      </Routes>
-    </>
+    <div>
+      <Quotes />
+      <Calculator />
+    </div>
```

Comment:
- [ ] You could place the Quotes component below the Calculator so the quote is displayed below the calculator.

Reply:
done 😃